app/reducers/messages.js

```diff
@@ -0,0 +1,12 @@
+import {List} from 'immutable';
+
+const EMPTY_LIST = List();
+
+export function messages(state = EMPTY_LIST, action) {
+  switch (action.type) {
+  case 'ADD_MESSAGE':
+    return state.push(action.message);
```

Comment:
We'll use this for better querying of stores https://github.com/faassen/reselect